spec/support/feature_helpers/publish_pages.rb

```diff
@@ -195,5 +195,9 @@ def terms_and_conditions_page
     def rollover_page
       @rollover_page ||= PageObjects::Publish::Rollover.new
     end
+
+    def rollover_recruitment_page
```

Comment:
That's because it is 😂 . If the the page objects were named in a consistent format we could do some meta-programming to generate methods for each one but that might be a task for another day. I'll card it up